libs/vm/include/vm/object.hpp

```diff
@@ -42,42 +42,44 @@ class Address;
 struct String;
 
 template <typename T>
-using IsPrimitive =
-    type_util::IsAnyOf<T, void, bool, int8_t, uint8_t, int16_t, uint16_t, int32_t, uint32_t,
-                       int64_t, uint64_t, float, double, fixed_point::fp32_t, fixed_point::fp64_t>;
+static constexpr bool IsPrimitive =
+    type_util::IsAnyOfV<T, void, bool, int8_t, uint8_t, int16_t, uint16_t, int32_t, uint32_t,
+                        int64_t, uint64_t, float, double, fixed_point::fp32_t, fixed_point::fp64_t>;
 
 template <typename T, typename R = void>
-using IfIsPrimitive = std::enable_if_t<IsPrimitive<std::decay_t<T>>::value, R>;
+using IfIsPrimitive = std::enable_if_t<IsPrimitive<std::decay_t<T>>, R>;
 
 template <typename T>
-using IsObject = std::is_base_of<Object, T>;
+static constexpr bool IsObject = std::is_base_of<Object, T>::value;
 
 template <typename T>
-struct IsPtr : std::false_type
+struct IsPtrImpl : std::false_type
 {
 };
 template <typename T>
-struct IsPtr<Ptr<T>> : std::true_type
+struct IsPtrImpl<Ptr<T>> : std::true_type
 {
 };
 
+template <typename T>
+static constexpr bool IsPtr = IsPtrImpl<T>::value;
```

Comment:
I had been wondering why had all the `::value` disappeared from the previous files in the review 😂 I like how this change makes other calls less cluttered.